README.md

```diff
@@ -1,77 +1,189 @@
 # packit.dev
 
-[packit.dev](https://packit.dev) website content
+Repository containing documentation and GH Actions used for deployment of the
+[packit.dev] website.
 
-## Clone
+### Installation
 
-This repository uses git submodules, so you have to `git clone --recurse-submodules` it.
-If you forget and realize later, just run:
+```
+$ yarn
+```
```

Comment:
I am trying to work with it locally.
I didn't realize immediately that `yarn` is not a `dnf` package. I installed it through `npm` but I was wondering if this is our preferred way? Maybe we can write it down.

Reply:
Noted, will both the docs and makefile targets as we have with dashboard.

Reply:
Fixed with the next point.